libcloudforensics/aws.py

```diff
@@ -465,6 +480,72 @@ def GetAccountInformation(self, info):
       raise KeyError('Key must be one of ["UserId", "Account", "Arn"]')
     return account_information.get(info)
 
+  def CreateSharedKMSKey(self, aws_account_id):
```

Comment:
I would tend to separate the logic in Creation and sharing of the key. Maybe make the second parameter optional? Or split this into two functions (one for creating and one for sharing?)